### DISCUSSION TITLE
note/application-and-practice-of-jwt-in-node/

### DISCUSSION THREAD

**Discussion by giscus** [gitguanqi/gitguanqi.github.io#14] [Announcements]
# note/application-and-practice-of-jwt-in-node/

导语：由于http是无状态的，请求响应过程中不存储记录用户身份信息，所以就出现了很多用户识别存储用户身份的方法，比如cookie，session，jwt。我最近做的一个接口服务使用了jwt来存储管理用户信息，相较于本地cookie存储，服务器端session存储，jwt就变得比较安全和节省方便，本文就jwt在node服务中的使用方法做一个简单的总结。

目录

jwt简介
+ ...

https://guanqi.xyz/note/application-and-practice-of-jwt-in-node/

**Comment by gitguanqi** [comment]
写的挺好的，学习到了！
